src/net/atlas/db.rs

```diff
@@ -51,7 +52,8 @@ const ATLASDB_INITIAL_SCHEMA: &'static [&'static str] = &[
         block_height INTEGER NOT NULL,
         is_available INTEGER NOT NULL,
         metadata TEXT NOT NULL,
-        contract_id STRING NOT NULL
+        contract_id STRING NOT NULL,
+        tx_id STRING NOT NULL
```

Comment:
I think this schema change should be applied with a migration. Otherwise, this would require a chainstate wipe, right?

Reply:
Right now, Atlas is disabled so not a problem, this table is empty for all the nodes with a version >= 2.0.4. 
If we release Atlas with the **boot from working dir** feature, we won't need a migration. If we're not, I'll probably open a PR that just update the schema, so we can avoid a migration once we release Atlas.

Reply:
Even though the table is empty, it is still created in versions > 2.0.4, which would still mean that the schema would be broken.

Reply:
Yes you're right. What I'm saying is that knowing that we have https://github.com/blockstack/stacks-blockchain/pull/2421 in the pipe, I think that we could and probably should try to avoid writing a migration by coordinating the release of these 2 changes.